mousestyles/path_diversity/path_features_advanced_proposed.py

```diff
@@ -0,0 +1,71 @@
+from future import print_function, absolute_import, division
+import numpy as np
+import pandas as pd
+from mousestyles.path_diversity.path_features import angle_between
+from mousestyles.path_diversity import path_index
+
+def compute_advanced(path_obj):
+    r"""
+    Return dictionary containing the radius, area of the rectangle 
+    spanned by the path, and absolute distance between start and end
+    points of each path.
```

Comment:
Currently we are not really sure which features will show significant difference over the strains. So it may make sense to include all features (including center angles and area covered in path) here and then to include only useful features we find interesting in plotting?